src/editor.js

```diff
@@ -264,9 +275,77 @@ const ContentPanel = ({
         codeMirrorElement is a promise resolved by codemirror(), hence the
         until directive. Once resolved, content can be empty.
       -->
+    ${TemplateLightbox({isTemplateLightboxDisplayed})}
     ${until(codeMirrorElement || Textarea({content}))}
   </div>
 `;
+const dispatchLoadHtmlFile = redispatchAs(g('load-html'));
+
+const TemplateLightbox = ({isTemplateLightboxDisplayed}) => html`
+  <div
+    class="${n('template-lightbox')}"
+    ?hidden=${!isTemplateLightboxDisplayed}
+  >
+    ${getNamesOfTemplates()}
+  </div>
+`;
+
+function getNamesOfTemplates() {
+  return html`
+    <div class="${g('flex-center')}">
+      <div
+        class="${n('template')}"
+        @click=${dispatchLoadHtmlFile}
+        data-name="app-install-ads"
+      >
+        <img width="100%" src="/static/app_install.jpg" />
+      </div>
+      <div
+        class="${n('template')}"
+        @click=${dispatchLoadHtmlFile}
+        data-name="multi-image-ads"
+      >
+        <video
+          width="100%"
+          autoplay
```

Comment:
This kills the CPU.

![killcrab](https://user-images.githubusercontent.com/254946/60469312-17d93a00-9c11-11e9-9f64-7ea7e1594e18.jpg)

1. Remove `autoplay`.
2. `play` and `pause` when the `templateLightbox` (also rename to `templatePanel`, since it's not a lightbox anymore) is toggled (call it when you change the `state` var).

```js
playResumeOnTemplatesToggle_() {
  const {isFilesPanelDisplayed} = this.state_;
  const videos = this.parent_.querySelectorAll(s('.template video'));
  for (const video of videos) {
    try {
      if (isFilesPanelDisplayed) {
        video.play();
      } else {
        video.pause();
      }
    } catch (_) {
      // Sometimes the browser yells at you when dynamically playing.
      // We don't care.
    }
  }
}
```